### ISSUE TITLE
before(nil) and after(nil)

### ISSUE THREAD

**Issue by zacharywelch** [radar/by_star#82]
Similar to #60 and #63, wondering if there's a way for `.before` and `.after` to return scope (do nothing) when the argument is nil, similar to `.between_times`.

> `between_times(start_time, nil)` - same as `after(start_time)`
    `between_times(nil, end_time)` - same as `before(end_time)`

Current behavior is to default base and time range scopes to [Time.zone.now](https://github.com/radar/by_star/blob/master/lib/by_star/base.rb#L49), but would prefer that only apply to the time range scopes 😅 

**Current behavior**
```ruby
Event.after(nil) #=> returns all events after the current time 
```

**Desired behavior**
```ruby
Event.after(nil) #=> returns all events
```

**Comment by radar** [comment]
I agree that both of these should return the default scope. Please submit a patch to that effect.

**Comment by johnnyshields** [comment]
Hmmm... it's ok but please make sure `.after_now` continues to work.
